cirq/testing/consistent_protocols.py

```diff
@@ -52,6 +55,7 @@ def assert_eigengate_implements_consistent_protocols(
             0, 1, -1, 0.5, 0.25, -0.5, 0.1, value.Symbol('s')),
         global_shifts: Sequence[float] = (0, 0.5, -0.5, 0.1),
         qubit_count: Optional[int] = None,
+        is_global_phase_sensitive: bool=False,
```

Comment:
I think we agreed in #1161 that global phase should always be preserved, so you should not be adding this flag.

Reply:
Long term we want global phase to be preserved, but short term it makes sense to have this.

Rename to `ignoring_global_phase` defaulting to false. We want to be able to find tests opting into global phase insensitivity and update them as we fix individual issues.

Reply:
Renamed.